<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginRollup</class>
 <widget class="QWidget" name="PluginRollup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>205</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="NodeName_Label">
     <property name="text">
      <string>Uninitialized</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="MaxSDK::QmaxSpinBox" name="spinBox"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MaxSDK::QmaxSpinBox</class>
   <extends>QSpinBox</extends>
   <header location="global">Qt/QmaxSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
